chapters/06_agent_based_extension/01_state_dependent/main.tex

```diff
@@ -0,0 +1,148 @@
+\section{State-dependent variation}\label{sec:state_dependent}
+
+This section focuses on creating a state-dependent variation of the queueing
+system described in Section~\ref{sec:queueing_section}.
+The state-dependent version is based on the idea that the speed of the servers
+could be based on the number of individuals present in the system.
+This was implemented only on the Discrete Event Simulation (DES) version of
+the queueing model described in Section~\ref{sec:discrete_event_simulation}.
```

Comment:
Does this need to be here? At the moment you're just describing the conceptual model right? The implementation doesn't matter.

Reply:
Yeah, I see. I think I mention it in the implementation section later on